_posts/en/newsletters/2025-01-31-newsletter.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 'Bitcoin Optech Newsletter #339'
+permalink: /en/newsletters/2025/01/31/
+name: 2025-01-31-newsletter
+slug: 2025-01-31-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Vulnerability in LDK claim processing:** Matt Morehouse
+  [posted][morehouse ldkclaim] to Delving Bitcoin to disclose an
+  vulnerability affecting LDK that he [responsibly disclosed][topic
+  responsible disclosures] and which was fixed in LDK version 0.1.  When
+  a channel is unilaterally closed with multiple pending [HTLCs][topic
+  htlc], LDK will attempt to resolve as many HTLCs as possible in the
+  same transaction to save on transaction fees.  However, if the channel
+  counterparty is able confirm any of the batched HTLCs first, that will
+  _conflict_ with the batch transaction and make it invalid.  In that
+  case, LDK would correctly create an updated batch transaction with the
+  conflict removed.  Unfortunately, if the counterparty's transaction
+  conflicts with multiple separate batches, LDK would incorrectly only
+  update the first batch.  The remaining batches would not be able to
+  confirm.
+
+  Some HTLCs need to be resolved before a deadline or the counterparty
```

Comment:
```suggestion
  Some HTLCs need to resolve before a deadline or the counterparty
```
Not completely sure about this suggestion, but less passive sounds better to me.